cmd/eval-dev-quality/cmd/report.go

```diff
@@ -80,6 +81,30 @@ func (command *Report) Execute(args []string) (err error) {
 		command.logger.Panicf("ERROR: %s", err)
 	}
 
+	// Create a CSV file that holds the models meta information.
+	var modelsMetaInformationCSVFile *os.File
+	if modelsMetaInformationCSVFile, err = os.OpenFile(filepath.Join(command.ResultPath, "meta.csv"), os.O_CREATE|os.O_EXCL|os.O_WRONLY, 0755); err != nil {
+		command.logger.Panicf("ERROR: %s", err)
+	}
+	defer modelsMetaInformationCSVFile.Close()
+
+	// Fetch all models meta information.
```

Comment:
Docstring that currently only openrouter supports querying meta information. Maybe even make this more generic by  just going through all providers right away? But also that can happen in the future.

Reply:
Updated the docstring. I think we can just query the openrouter for now since it is the only one that has meta information.